Source/WebKit/WebProcess/WebPage/FindController.cpp

```diff
@@ -299,22 +305,11 @@ void FindController::findString(const String& string, OptionSet<FindOptions> opt
                 m_foundStringMatchIndex++;
         }
     }
-#if ENABLE(IMAGE_ANALYSIS)
-    if (canTriggerImageAnalysis == TriggerImageAnalysis::Yes) {
-        m_webPage->corePage()->analyzeImagesForFindInPage([weakPage = WeakPtr { m_webPage }, string, options, maxMatchCount] {
-            if (weakPage)
-                weakPage->findController().findString(string, options, maxMatchCount, TriggerImageAnalysis::No); 
-        });
-    }
-#endif
 
```

Comment:
Technically this regresses the internal feature `imageAnalysisDuringFindInPage` because now you have to wait for image analysis to complete before getting any text matches, but I think this is okay because the increased latency will only be noticeable on pages with a lot of images, and the feature is off by default with no plans to turn it on.